fs/fcb/src/fcb.c

```diff
@@ -265,3 +301,183 @@ fcb_clear(struct fcb *fcb)
     }
     return rc;
 }
+
+int
+fcb_init_flash_area(struct fcb *fcb, int flash_area_id, uint8_t version)
```

Comment:
Is this function needed?  I don't see it getting called anywhere, and I don't see it declared in any headers.  If it is needed, then it should be declared in a header, and the references to a "log" should be removed from the comments.